internal/multistorage/cache/alive_check.go

```diff
@@ -0,0 +1,74 @@
+package cache
```

Comment:
Please use `git mv` next time you'll move the files to produce actual file diffs.